docs/sagas.md

```diff
@@ -0,0 +1,239 @@
+---
+id: sagas
+title: Sagas
+---
+
+A saga describes a long running process as a sequence of events.
+
+## Sagas Overview
+
+You can define a saga as a set of event handler functions. Each such function runs in response to a specific event and can do the following:
+
+- Send a command to an aggregate
+- Schedule a command on the specified moment in time
+- Store intermediate data in a persistent storage
+- Trigger a side effect
+
+This allows you to organize branching chains of events and side effects to describe processes of any complexity. For example, the code below demonstrates a saga that organizes a web site's user registration process:
```

Comment:
What is "This" (at the beginning of the sentence)? Does it refer to "Saga"?
Change "branching" to "branched".
Instead of "organizes" can we say "structures"?

Reply:
> Change "branching" to "branched".

done

Reply:
> Instead of "organizes" can we say "structures"?

I think, we can. Changed to "structures"

Reply:
Okay, let's change it to "This functionality allows..." - as you mentioned.

Reply:
done